### PR TITLE
VDS-873, VDS-906: Assign user group to a customer/organization

### PR DESCRIPTION
**Pull request by ilyawzrd**

### CODE REVIEW

File: src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/resources/userGroups.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+angular.module('virtoCommerce.DemoSolutionFeaturesModule')
+    .factory('virtoCommerce.DemoSolutionFeaturesModule.userGroupsApi', ['$resource', function ($resource) {
+        return $resource('api/demo/members', {}, {
+            taggedMember: { url: 'api/demo/members/tagged/:id', method: 'GET' },
```
Comment (pushnitsa) [inline src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/resources/userGroups.js]: Just get

File: src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/module.js [hunk 2]
```diff
@@ -41,6 +41,22 @@ angular.module(moduleName, [])
                 }, 'customerOrderDetailWidgets');
             });
 
+            featureManager.isFeatureEnabled('UserGroupsInheritance').then(() => {
+                widgetService.registerWidget({
+                    isVisible: function (blade) { return blade.controller === 'virtoCommerce.customerModule.memberDetailController'; },
```
Comment (pushnitsa) [inline src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/module.js]: Why we need this condition?

File: src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/user-groups-list.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                                    const newTags = _.difference(entity.tags, blade.currentEntity.tags);
+                                    blade.currentEntity.tags = [...blade.currentEntity.tags, ...newTags];
+                                    bladeNavigationService.closeChildrenBlades(blade);
+                                    saveChanges();
+                                }
+                            };
+                            bladeNavigationService.showBlade(newBlade, blade);
+                        });
+                    },
+                    canExecuteMethod: () => true
```
Comment (pushnitsa) [inline src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/user-groups-list.js]: Maybe we should use customer permission to manipulate member tags?

**Review by pushnitsa — CHANGES_REQUESTED** [review]

File: src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/module.js (hunk 2 above)
Comment (ilyawzrd) [reply src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/module.js]: that's how we check if we're on the needed blade

File: src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/resources/userGroups.js (hunk 1 above)
Comment (ilyawzrd) [reply src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/resources/userGroups.js]: done

File: src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/user-groups-list.js (hunk 3 above)
Comment (ilyawzrd) [reply src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/user-groups-list.js]: done

File: src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/module.js (hunk 2 above)
Comment (ilyawzrd) [reply src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/module.js]: done, removed

**Review by pushnitsa — APPROVED** [review]

**Comment by ilyawzrd** [comment]
Closed, all changes are in https://github.com/VirtoCommerce/vc-module-demo-features/pull/71

**Pull request closed without merging**